spec/requests/liquid_tags_request_spec.rb

```diff
@@ -0,0 +1,27 @@
+require 'rails_helper'
+
+RSpec.describe "LiquidTags", type: :request do
+  describe "GET /liquid_tags" do
+    context "when not signed in do" do
+      it "returns a list of all custom Liquid tags" do
+        get liquid_tags_path
+
+        expect(response.status).to eq 401
+      end
+    end
+
+    context "when signed in" do
+      let(:user) { create(:user) }
+      before { sign_in(user) }
+
+      it "returns an array of all custom Liquid tags", :aggregate_failures do
+        get liquid_tags_path
+
+        expect(response.status).to eq 200
```

Comment:
you can use `expect(response).to have_http_status(:ok)`

Reply:
Ah, yes. I guess I'm jut used to the codes, but this is probably nicer for many people.